### ISSUE TITLE
Crawler does not start in Windows [Bug] 

### ISSUE THREAD

**Issue by athena9** [Chia-Network/chia-blockchain#10798]
### What happened?

This was reported on Keybase by a few people at 1.3.0 release. Looks like a fix didn't make the 1.3.1 cut and I can't find an open issue. Opening one so it can be tracked.

chia start crawler

### Version

1.3.1

### What platform are you using?

Windows

### What ui mode are you using?

CLI

### Relevant log output

```shell
2022-03-18T13:40:34.874 daemon __main__                   : ERROR    problem starting chia_crawler
Traceback (most recent call last):
  File "Lib\site-packages\chia\daemon\server.py", line 1108, in start_service
  File "Lib\site-packages\chia\daemon\server.py", line 1299, in launch_service
  File "subprocess.py", line 951, in __init__
  File "subprocess.py", line 1420, in _execute_child
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

**Comment by wallentx** [comment]
Can you try to install from source to see if this works? We will look at our installer to make sure everything is getting built correctly.

**Comment by paninaro** [comment]
Looks like crawler and seeder are not included in `pyinstaller.spec`, so the setup scripts aren't being generated for them.

**Comment by emlowe** [comment]
This was resolved in 1.3.5 I believe - certainly at some point
